influxdb-mixin/dashboards/influxdb-instance-overview.libsonnet

```diff
@@ -0,0 +1,1881 @@
+      .addTemplates(
+        [
+          template.datasource(
+            promDatasourceName,
+            'prometheus',
+            null,
+            label='Data Source',
+            refresh='load'
+          ),
+          template.new(
+            'job',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds,job)',
+            label='Job',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=0
+          ),
+          template.new(
+            'cluster',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds{%(multiclusterSelector)s}, cluster)' % $._config,
+            label='Cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.*',
+            hide=if $._config.enableMultiCluster then '' else 'variable',
+            sort=0
+          ),
+          template.new(
+            'influxdb_cluster',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds{%(influxdbSelector)s}, influxdb_cluster)' % $._config,
+            label='InfluxDB cluster',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='',
+            sort=0
+          ),
+          template.new(
+            'instance',
+            promDatasource,
+            'label_values(influxdb_uptime_seconds{%(influxdbSelector)s, influxdb_cluster=~"$influxdb_cluster"}, instance)' % $._config,
+            label='Instance',
+            refresh=2,
+            includeAll=true,
+            multi=true,
+            allValues='.+',
+            sort=0
+          ),
+        ]
+      )
+      .addPanels(
+        [
+          uptimePanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 0, y: 0 } },
+          bucketsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 3, y: 0 } },
+          usersPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 6, y: 0 } },
+          replicationsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 9, y: 0 } },
+          remotesPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 12, y: 0 } },
+          scrapersPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 15, y: 0 } },
+          dashboardsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 18, y: 0 } },
+          threadsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 3, x: 21, y: 0 } },
+          goRow { gridPos: { h: 1, w: 24, x: 0, y: 8 } },
+          timeSinceLastGCPanel(getMatcher($._config)) { gridPos: { h: 8, w: 6, x: 0, y: 9 } },
+          gcTimePanel(getMatcher($._config)) { gridPos: { h: 8, w: 9, x: 6, y: 9 } },
+          gcCPUUsagePanel(getMatcher($._config)) { gridPos: { h: 8, w: 9, x: 15, y: 9 } },
+          heapMemoryUsagePanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 0, y: 17 } },
+          goThreadsPanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 12, y: 17 } },
```

Comment:
Personal take, I'd consider the Go stats quite unimportant and would move them to the bottom of the dashboard.

@rgeyer what are your thoughts here?

Reply:
Yes, I'd agree. Go process details are tangentially useful, but should not be prioritized.